vectordatasource/transform.py

```diff
@@ -3166,8 +3169,8 @@ def keep_n_features_gridded(ctx):
         return None
 
     minx, miny, maxx, maxy = ctx.unpadded_bounds
```

Comment:
By here do we know if we're in a 256 or 512 px tile and can adjust a `grid_width` or `grid_height` multiplier accordingly?

Reply:
No, there's no concept of pixel size here as far as I know. It's just processing a bounding box of vector data.

Reply:
wacky (future) idea: seems like you could even compute the features to keep in for multiple tile sizes and tag them as such.  We could include this stuff in our layer output but then weed it out in tilequeue where we know what size we're cutting.